<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>381</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>381</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="user_name">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove gray;border-radius:8px;padding:2px 4px</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_password">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove gray;border-radius:8px;padding:2px 4px</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signin">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove gray;border-radius:8px</string>
    </property>
    <property name="text">
     <string>登陆</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_up">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>180</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove gray;border-radius:8px</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
